telethon/events/chataction.py

```diff
@@ -113,7 +114,11 @@ def build(cls, update, others=None, self_id=None):
             elif isinstance(action, types.MessageActionPinMessage) and msg.reply_to:
                 return cls.Event(msg,
                                  pin_ids=[msg.reply_to_msg_id])
-
+            elif isinstance(action, types.MessageActionGameScore):
+                return cls.Event(msg,
+                                 game_id=action.game_id,
```

Comment:
The `Event` class does not have this parameter. Doesn't this fail?

Reply:
ah yes done